### DISCUSSION TITLE
Loading X509 certificate

### DISCUSSION THREAD

**Discussion by dev-area** [polhenarejos/pico-hsm#33] [Q&A] [answered]
Hi
is it possible to load X509 certificate to the HSM?
(and also read it)
thanks

**Comment by polhenarejos** [comment, marked as answer]
Sure. You can store any binary data up to 2048 kB.

https://www.picokeys.com/2022/07/11/store-binary-data/

**Reply by dev-area** [reply]
thanks. great feature
